hips/utils/wcs.py

```diff
@@ -4,16 +4,20 @@
 import numpy as np
 from astropy.coordinates import SkyCoord
 from astropy.wcs import WCS
+from astropy.coordinates import Angle
 from astropy.wcs.utils import pixel_to_skycoord, wcs_to_celestial_frame
 
 __all__ = [
     'WCSGeometry',
 ]
 
-__doctest_skip__ = ['WCSGeometry']
+__doctest_skip__ = [
+    'WCSGeometry',
+    'WCSGeometry.*'
```

Comment:
This is required to skip doctests for all the methods within `WCSGeometry`, in this case the `create_simple` `@classmethod`, more [here](http://docs.astropy.org/en/stable/development/testguide.html#skipping-doctests).